packages/sku/lib/suggestScript.js

```diff
@@ -1,14 +1,16 @@
-const exists = require('./exists');
+const { runCommand, isNpm, packageManager } = require('./packageManager');
+
 const chalk = require('chalk');
-const { getPathFromCwd, requireFromCwd } = require('./cwd');
+const { requireFromCwd } = require('./cwd');
 
 /**
- * @param {scriptContents} string
+ * @param {string} scriptContents
  */
 const findPackageScript = (scriptContents) => {
   let pkg;
   try {
-    pkg = requireFromCwd('package.json');
+    // CWD should be the root of the package
+    pkg = requireFromCwd('./package.json');
   } catch (err) {
     pkg = { scripts: {} };
```

Comment:
Resolving `package.json` from cwd (package root) would always fail, so the `catch` clause was executing every time, so scripts from `package.json` were never suggested.

Reply:
Good catch 🤭